dashboard/public/js/main.js

```diff
@@ -103,3 +103,69 @@ function firstClientVisit() {
 		return true;
 	}
 }
+
+var textBox = document.getElementById("password-box");
+var password = document.getElementById('password');
+
+function passwordStrength() {
+	var parentDiv = document.createElement("div");
+	
+	var meter = document.createElement("METER");
+	meter.id = "password-strength-meter";
+	meter.max = "4";
+	
+	var strengthText = document.createElement("p");
+	strengthText.id = "password-strength-text";
```

Comment:
Don't add extra code which is not required. I think this ID does not have any use in the code.